services/build-status.js

```diff
@@ -13,7 +13,13 @@ const greenStatuses = [
 
 const orangeStatuses = ['partially succeeded', 'unstable', 'timeout']
 
-const redStatuses = ['error', 'failed', 'failing', 'infrastructure_failure']
+const redStatuses = [
+  'error',
+  'failed',
+  'failing',
+  'failure',
```

Comment:
Thanks! Could you also update the corresponding test in `build-status.spec.js`?

Reply:
My bad! Will do it now!

Reply:
Done! 🚀